src/int.cr

```diff
@@ -486,6 +486,51 @@ struct Int
     io << type
   end
 
+  # Prints this integer as a binary value in a human-readable format using
+  # multiples of 1024 as prefixes.
+  #
+  # Values with binary measurements such as computer storage (e.g. RAM size) are
+  # typically expressed using unit prefixes based on 1024 (instead of multiples
+  # of 1000 as per SI standard). This method by default uses an extended range
+  # of the JEDEC units (`K`, `M`, `G`, `T`, `P`, `E`, `Z`, `Y`) which equals to
+  # the prefixes of the SI system except for uppercase `K` and is based on
+  # powers of 1024. The IEC standard prefixes (`ki`, `Mi`, `Gi`, `Ti`, `Pi`,
+  # `Ei`, `Zi`, `Yi`) will be used if *standard* is `:IEC`.
+  #
+  # ```
+  # 1.humanize_bytes                          # => "1B"
+  # 1024.humanize_bytes                       # => "1.0KB"
+  # 1536.humanize_bytes                       # => "1.5KB"
+  # 524288.humanize_bytes                     # => "512KB"
+  # 1073741824.humanize_bytes(standard: :IEC) # => "1.0GiB"
+  # ```
+  #
+  # See `Number#humanize` for more details on the behaviour and arguments.
+  def humanize_bytes(io : IO, precision : Int = 3, separator = '.', *, significant : Bool = true, standard : Symbol = :JEDEC) : Nil
```

Comment:
IEC should be the default :)

Reply:
IEC is still a long way from being widely adopted. People still mostly use JEDEC all over the place, so this should be the default (IMHO).

Reply:
Every *human* understands IEC, and it makes the fact that you're using base-1024 absolutely unambiguous.

Reply:
And not using IEC because it's not widely adopted is a massive self-fulfilling prophecy

Reply:
I agree. But still, I'm not convinced making IEC the default is the right path. 
OTOH, I really don't care much and it's easy to change.